src/Http/Controllers/PageController.php

```diff
@@ -17,7 +16,9 @@ class PageController extends Controller
      */
     public function home(Request $request): Application|Factory|View
     {
-        return view('cms::pages.home');
+        $page = static::getModelClassName(Resources::PAGE)::where('is_home', true)->first();
+
+        return view("cms::$page->title", compact('page'));
```

Comment:
If the title has a space or a strange character would this work?I guess using slug would make more sense